go-controller/pkg/allocator/pod/pod_annotation_test.go

```diff
@@ -161,11 +164,9 @@ func Test_allocatePodAnnotationWithRollback(t *testing.T) {
 			},
 			wantUpdatedPod: true,
 			wantPodAnnotation: &util.PodAnnotation{
-				IPs:  ovntest.MustParseIPNets("192.168.0.4/24"),
-				MAC:  util.IPAddrToHWAddr(ovntest.MustParseIPNets("192.168.0.4/24")[0].IP),
-				Role: types.NetworkRolePrimary,
```

Comment:
why did you remove the `Primary` thing ? was it to sync w/ the test's description ? 

If so, please, isolate it in a separate commit. It's a bit distracting here.

Reply:
nice catch, please see the first commit, i returned the primary, but added disableMultiNetwork for this test
since adding EnableMultiNetwork affected this test (it does add routes / gateway in case multi networks + primary is enabled)